ModeratorFrontEnd/AIForOrcas/AIForOrcas.DTO/API/Detections/DetectionQueryParameters.cs

```diff
@@ -24,11 +24,17 @@ public class DetectionQueryParameters
 		public string SortOrder { get; set; } = "desc";
 
 		/// <summary>
-		/// Timeframe for the record set (last 30m, 24h, 1w, 30d, all).
+		/// Timeframe for the record set (last 3h, 6h, 24h, 1w, 30d, all).
```

Comment:
you also have a 30 minute timeframe so you don't need to delete it from the comment.